### DISCUSSION TITLE
get Album by Track or PlaylistTrack

### DISCUSSION THREAD

**Discussion by antsam23** [spotify-web-api-java/spotify-web-api-java#337] [Q&A] [answered]
Hi! Is it possible to get Album by Track or PlaylistTrack? I see the json has the album field but I don't see public methods to get albums in those classes.

**Comment by dargmuesli** [comment, marked as answer]
You can get an [AlbumSimplified](https://spotify-web-api-java.github.io/spotify-web-api-java/se/michaelthelin/spotify/model_objects/specification/AlbumSimplified.html) through
1. [PlaylistTrack.html#getTrack()](https://spotify-web-api-java.github.io/spotify-web-api-java/se/michaelthelin/spotify/model_objects/specification/PlaylistTrack.html#getTrack())
2. [Track.html#getAlbum()](https://spotify-web-api-java.github.io/spotify-web-api-java/se/michaelthelin/spotify/model_objects/specification/Track.html#getAlbum())

**Reply by antsam23** [reply]
Oh I see :) Thank you very much!
